jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/SliderRange.java

```diff
@@ -14,7 +14,8 @@
   private final static String areaValueNow = "aria-valuenow";
   private final static String style = "style";
   private final static String ariaValueText = "aria-valuetext";
-  private int LocalLeft; //local variable for substitution instead of parameter reflectionSetValue to Avoid reassigning parameters
+  private int iOiginalValue; //auxiliary variable to keep original integer values;
+  private int Localvariable; //local variable for substitution instead of parameter reflectionSetValue to Avoid reassigning parameters
```

Comment:
Please use more understandable name, not just name with word "variable"
Start name from small letter, new word in name from capital letter